packages/components/datepicker/README.mdx

```diff
@@ -6,22 +6,20 @@ slug: /components/datepicker/
 github: 'https://github.com/contentful/forma-36/tree/main/packages/components/datepicker'
 storybook: 'https://v4-f36-storybook.netlify.app/?path=/story/components-datepicker'
 typescript: ./src/Datepicker.tsx
+section: 'dateComponents'
 ---
 
-Short component description covering when it's used.
-(Optional: give a most common example)
+import { Badge, Flex } from '@contentful/f36-component';
 
-Optional: Most common misuse and what to use instead.
+Datepicker component enables users to enter a date either through text input or selecting a date from the Calendar dropdown.
 
-Props comparison to our current datepickers:
+The Datepicker component combines text input, and calendar in dropdown.
 
-<!--
-Current     DayPicker
-value       selected
-onChange    onSelect
-min         fromDate
-max         toDate
--->
+Use Datepicker to ask user's for date input in forms, modals, or filtering.
+
+<Flex>
+  <Badge variant="featured">Beta</Badge>
+</Flex>
```

Comment:
Not sure how we usually handle this, but I think this would look nicer in the title. What do you think?

<img width="753" alt="Screenshot 2022-06-17 at 13 41 27" src="https://user-images.githubusercontent.com/219017/174291699-a8d68342-da57-4fc5-a270-bb2b02480b8b.png">

Reply:
If we add the note we wouldn't need the badge, if I remember correctly we didn't add the badge for the pagination component when it was added, only the note

Reply:
That works for me.

Reply:
Done, now it shows the alpha note